dbt/dbt_project.yml

```diff
@@ -61,3 +61,8 @@ models:
       +schema: qc
 tests:
   +schema: test_failure
+
+seeds:
+  athena:
+    location:
+      +schema: location
```

Comment:
It'd be nice if we could have seeds inherit the schema config from the models, so that we don't have to duplicate them. Perhaps we could rig something up with anchors and aliases? I didn't bother with it here, however.

Reply:
I think it's fine for it to be a little bit redundant!